plugin/driver.go

```diff
@@ -515,6 +515,8 @@ func (p *Plugin) setOptions(driverTaskConfig *drivers.TaskConfig) (*shim.Options
 		unveil = append(unveil, "rwxc:"+driverTaskConfig.Env["NOMAD_TASK_DIR"])
 		unveil = append(unveil, "rwxc:"+driverTaskConfig.Env["NOMAD_ALLOC_DIR"])
 		unveil = append(unveil, "rwxc:"+driverTaskConfig.Env["NOMAD_SECRETS_DIR"])
+		parent := filepath.Dir(driverTaskConfig.Env["NOMAD_TASK_DIR"])
+		unveil = append(unveil, "rwxc:"+filepath.Join(parent, "tmp"))
```

Comment:
This is probably ignorance on my part, but this surprises me; if the `tmp` dir is a child of the `NOMAD_TASK_DIR`, why isn't it automatically accessible by our unveiling the `NOMAD_TASKD_DIR`?

Reply:
:facepalm: nevermind... `parent` is the _parent_ of the `NOMAD_TASK_DIR`.

Reply:
yeah that caught me by surprise too, but I don't make the rules